app/controllers/advertisement_previews_controller.rb

```diff
@@ -30,13 +52,28 @@ def theme_name
     ENUMS::AD_THEMES[params[:theme]]
   end
 
+  def sponsor?
+    request.format == "svg"
```

Comment:
Will this always be true?

Reply:
Perhaps not always but this impl matches the advertisements controller. I'm fine leaving it hard coded for now.